CHANGELOG.md

```diff
@@ -121,6 +121,11 @@
 
 * Make `empty_count` auto-correctable.  
   [KS1019](https://github.com/KS1019/)
+  
+* Make `private_swiftui_state` auto-correctable. 
+  Access control has been made stricter and treats `fileprivate` as a violation.
```

Comment:
Please append two spaces to the end of the description. They are required to enforce a line break in the rendered Markdown:
```suggestion
* Make `private_swiftui_state` auto-correctable. Access control
  has been made stricter and treats `fileprivate` as a violation.  
```